src/host/ledger.h

```diff
@@ -899,6 +901,19 @@ namespace asynchost
     size_t write_entry(
       const uint8_t* data, size_t size, bool committable, bool force_chunk)
     {
+      auto header = serialized::peek<kv::SerialisedEntryHeader>(data, size);
+
+      bool force_chunk_in_header =
+        header.flags & kv::EntryFlags::FORCE_LEDGER_CHUNK;
+
+      if (force_chunk_in_header)
```

Comment:
Should this throw if `force_chunk_in_header` but `committable` is `false` since it would break the invariants that all ledger chunks should end on a committable entry (i.e. signature)?

Reply:
Good point, yes it should!